sdk/src/config/MAINNET.ts

```diff
@@ -247,6 +248,17 @@ const MAINNET: { [chain in MainnetChainName]: ChainConfig } = {
     nativeTokenDecimals: 18,
     cctpDomain: 6,
   },
+  klaytn: {
+    key: 'klaytn',
+    id: 13,
+    context: Context.ETH,
+    contracts: {
+      ...CONTRACTS.MAINNET.klaytn,
+      relayer: '0x27428DD2d3DD32A4D7f7C497eAaa23130d894911',
```

Comment:
Is there actually xLabs relayer support on Klaytn?

Reply:
I added it because I saw it in the docs https://docs.wormhole.com/wormhole/blockchain-environments/evm#mainnet-contracts-8217

Reply:
on the xlabs side I didn't investigate, I will check

Reply:
we don't have an xlabs relayer yet, so I deleted it from the relayer field